python/apps/taiga/src/taiga/projects/projects/api/__init__.py

```diff
@@ -247,6 +247,6 @@ async def list_my_project_permissions(id: B64UUID, request: AuthRequest) -> list
 async def get_project_or_404(id: UUID) -> Project:
     project = await projects_services.get_project(id=id)
     if project is None:
-        raise ex.NotFoundError(f"Project {id} does not exist")
+        raise ex.NotFoundError("Project does not exist")
```

Comment:
Why do you change that?

Reply:
I can't recover the original Id from a b64ID (as there's a one to many correspondence), so one solution is to avoid returning it.